docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,2005 @@
+{
```

Comment:
<p>I actually don't think that we should allow a range of window sizes and, instead, only allow <code>m</code> as an input. However, we add a new parameter to this function called <code>r=None</code>  which is basically your <code>r_init</code>.  I want the user-facing parameter to be called <code>r</code> . Then, in the notebook (not in the <code>discords</code> function) , we can manually demonstrate how to use the discords function to loop over a range of window sizes. In other words, I only want <code>discords</code> to return the top k discords for a single value of <code>m</code> .  If the user wants the discords over a range of <code>m</code> then it should be easy to accomplish this outside of the <code>discords</code> function. Technically, I don't believe that there is any special performance gain, right?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
> Technically, I don't believe that there is any special performance gain, right?

Yes...the only thing function `_discords` does is that it uses discord of length `m` to find discords of length `m+1`. That is its only advantage. However, again, that would be just initial guess for `m+1`, and not necessarily a good value. But, we assume it is good enough.... 

So...as you said, I think there is no special performance gain here... and we may add note to the function  docstring to let user know they can use discord distance of length `m` as value of parameter `r` when discovering discords of length `m+1`. 

So...yeah... it seems reasonable to let users do it themselves.

Reply:
<p>I think we just show an example in this notebook where we get the top discords by calling the <code>discords</code> function for a range of <code>m</code></p><p> </p><p>So, after we move all of these functions into <code>stumpy/discords.py</code> , we can call the <code>discords</code> function for a range of <code>m</code>    </p><p> </p>